async-opcua-types/src/relative_path.rs

```diff
@@ -340,21 +382,6 @@ impl RelativePathElement {
     }
 }
 
-#[derive(Debug, Clone)]
-/// Error returned from parsing a relative path.
-pub enum RelativePathError {
```

Comment:
Could you, instead of removing this error, just imlement `From<RelativePathError> for OpcUaError`? I don't see a good reason to flatten the error now.

Do feel free to derive `Error` on `RelativePathError`, just using the docstrings as error messages.

Reply:
yes might be. I was unsure what the correct approach is here. I just proposed something.
And yes maybe `From<RelativePathError> for OpcUaError` is a good solution. 
I will change that